src/api/app/controllers/webui/webui_controller.rb

```diff
@@ -231,7 +231,8 @@ def set_influxdb_data
     InfluxDB::Rails.current.tags = {
       beta: User.possibly_nobody.in_beta?,
       anonymous: !User.session,
-      interface: :webui
+      interface: :webui,
+      bs_request_action_type: nil
```

Comment:
```suggestion
      interface: :webui
```
I would leave this lines unchanged. Why is it needed to assign `bs_request_action_type` to `nil` here?

Reply:
we completely changed this PR. please re-review...